epoxy-processor/src/main/java/com/airbnb/epoxy/ClassToGenerateInfo.java

```diff
@@ -203,5 +225,33 @@ public MethodInfo(String name, Set<Modifier> modifiers, List<ParameterSpec> para
       this.modifiers = modifiers;
       this.params = params;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
I like that idea, but then we probably should do the same with `ConstructorInfo` since they are essentially the same, but this one is rather small.